index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=inter">
+    
+      
+</head>
+<body>
+    <header class="navbar">
+        <a class="welcome">Welcome</a>
+        <img class="menu" alt="button" src="Enabled.png" >
+    </header>
+    <div class="main">
```

Comment:
- [ ] make sure to use ```section``` tag instead of ```div```, please implement this as the previous reviewer suggested